.github/workflows/test.yml

```diff
@@ -34,15 +34,16 @@ jobs:
     # We test on all of the maintained versions: https://nodejs.org/en/about/releases/
     strategy:
       matrix:
-        node-version: ['16.x', '18.x', '20.x', '21.x']
+        node-version: ["lts/-1", "lts/*", "current"]
```

Comment:
Why this change? 🤔 Engine states that we should support node 14, so we should test it on node14 as well?

Reply:
Esbuild doesn't support it. There are ways around it but it didn't feel worth the investment for a node version that no longer receives security patches and thus shouldn't be used by anyone in production 😅
At the same time we aren't intentionally breaking node 14 support in userland so it didn't feel right to bump engines 🤷

Reply:
Right. Is there a way we can still test it with node 14 though? Now we might break the engine support without knowing